bigchaindb/pipelines/README.md

```diff
@@ -0,0 +1,39 @@
+# Overview
+
+Code in this module concerns the long-running processes of BigchainDB.  Some are run as single processes while others may run as many processes in parallel.  When changes are detected in the `backlog`, `block`, or `votes` tables, they are handled here.  Everything is started in [`processes.py`](https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/processes.py).
+
+All the classes defined in these files depend heavily on the the [`multipipes`](https://github.com/bigchaindb/multipipes/) library.  Each contains a static method `create_pipeline` which describes how the `Pipeline` is set up. Consider `votes.py`:
```

Comment:
[the the](https://en.wikipedia.org/wiki/The_The) are an English musical group that have been active in various forms since 1979, ...